examples/metrics/kafka-mirror-maker-2-metrics.yaml

```diff
@@ -33,9 +33,21 @@ spec:
           checkpoints.topic.replication.factor: 1
       topicsPattern: ".*"
       groupsPattern: ".*"
-  metrics:
-    # Inspired by kafka-connect rules
-    # https://github.com/prometheus/jmx_exporter/blob/master/example_configs/kafka-connect.yml
+  metricsConfig:
+    type: jmxPrometheusExporter
+    valueFrom:
+      configMapKeyRef:
+        name: mirror-maker-2-metrics
+        key: metrics-config.yml
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: mirror-maker-2-metrics
+  labels:
+    app: strimzi
+data:
+  metrics-config.yml: |
```

Comment:
link

Reply:
Links to Strimzi doc? What about downstream equivalents?

Reply:
@laidan6000 ^

Reply:
They can be changed downstream. But I'm not sure we have any Strimzi docs explaining the details of the configuration. So maybe this should be link to JMX Prometheus docs.